src/Sarif/Autogenerated/InvocationEqualityComparer.cs

```diff
@@ -10,10 +10,10 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// <summary>
     /// Defines methods to support the comparison of objects of type Invocation for equality.
     /// </summary>
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.20.0.0")]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.21.0.0")]
     public sealed class InvocationEqualityComparer : IEqualityComparer<Invocation>
     {
-        public static readonly InvocationEqualityComparer Instance = new InvocationEqualityComparer();
+        internal static readonly InvocationEqualityComparer Instance = new InvocationEqualityComparer();
```

Comment:
I suggest making the whole concrete comparer classes internal rather than just the Instance properties.